build-logic/src/main/kotlin/io/github/droidkaigi/confsched2022/primitive/AndroidGradleDsl.kt

```diff
@@ -28,6 +28,9 @@ fun Project.setupAndroid() {
 
         defaultConfig {
             minSdk = 23
+
+            versionCode = 1
+            versionName = "1.0.0"
```

Comment:
I added version name & version code in feature plugin's default config to show app version name on about screen